site/en/docs/extensions/mv3/known-issues/index.md

```diff
@@ -54,9 +54,9 @@ This section lists major features that will be added to the Manifest V3 platform
 
 ### Increased quota for session storage in the Storage API {: #increased-session-storage-quota }
 
-**Summary:** When it was introduced, the `session` storage area had an intentionally conservative maximum quota of 1 MB. We are planning to increase this limit, but have not yet settled on a new value.
+**Summary:** When it was introduced, the `session` storage area had an intentionally conservative maximum quota of 1 MB. This limit is being increased to 10 MB.
 
-**Estimated timeline**: Targeting Canary support in the first quarter of 2023.
+**Estimated timeline**: This will be available in Chrome 112.
```

Comment:
```suggestion
**Estimated timeline**: This will be available in Chrome 112. Chrome 112 beta is planned around March 9, with a stable release around the beginning of April.
```

Reply:
The reason I'm avoiding definite language for the release dates is that they're not guaranteed. The release time for stable is always on a Tuesday, but the time varies. The release time for beta is always two days later at around 11:00 Pacific. Delays are not unheard of. I once saw a beta that didn't make it out until the following Monday. https://chromiumdash.appspot.com/schedule